### ISSUE TITLE
Allow disabling keyboard navigation for Tabsets

### ISSUE THREAD

**Issue by vdurante** [valor-software/ng2-bootstrap#5628]
**Is your feature request related to a problem? Please describe.**

I am developing an application that has Tabs that work similarly to Excel Sheets. If you double click on the Tab name, it becomes an input field you can edit that modifies the name of the tab.

The issue is: since the Tabset has keyboard navigation, it conflicts with keyboard input on my element. I am unable to use space, home, arrows, etc.

**Describe the solution you'd like**

I would like a direct capable of disabling/enabling the keyboard navigation.

**Additional context**

The element:
![image](https://user-images.githubusercontent.com/5401232/73398037-7360c880-42c3-11ea-97d1-31b58d2c9445.png)

Sample code:
```html
<tabset>
    <tab>
    <ng-template tabHeading>
        <span>Tab 1</span>
    </ng-template>
    Tab 1 content
    </tab>
    <tab>
    <ng-template tabHeading>
        <span><input type="text" autofocus value="Tab 2"/></span>
    </ng-template>
    Tab 2 content
    </tab>
</tabset>
```

**Comment by Beni312** [comment]
similar to #5538
